### DISCUSSION TITLE
Support for Windows Update?

### DISCUSSION THREAD

**Discussion by T3rr0** [microsoft/openjdk#46] [Q&A] [answered]
Can’t find any information how Microsoft is gonna provide automatic updates for its Java. Would be awesome if you'd offer updates for eg Windows x64 OpenJDK via WU. Manually updating different builds of OpenJDKs is challenging

**Comment by karianna** [comment, marked as answer]
Please see https://github.com/microsoft/openjdk/discussions/40 - we'll be providing updates via Winget and are working on the default upgrade behaviour of the Windows installer in particular.
